src/App.tsx

```diff
@@ -1,24 +1,138 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  ChangeEvent,
+  useEffect,
+  useState,
+} from 'react';
+
+import { getTodos, addTodo, removeTodo } from './api/todos';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { Filter } from './components/Filter/Filter';
+import { NewTodo } from './components/NewTodo/NewTodo';
+import NotificationError from
+  './components/NotificationError/NotificationError';
 
-const USER_ID = 0;
+const USER_ID = 6846;
 
 export const App: React.FC = () => {
+  const [task, setTask] = useState('');
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState('');
+  const [tempTodo, setTemptodo] = useState<Todo | null>(null);
+  const [isDataUpdated, setIsdataUpdated] = useState(false);
+  const [activeIds, setActiveIds] = useState([0]);
+
+  const handleTodoChange = (event: ChangeEvent<HTMLInputElement>) => {
+    setTask(event.target.value);
+  };
+
+  const loadTodos = async () => {
+    setIsdataUpdated(true);
+
+    try {
+      const todosFromServer = await getTodos(Number(USER_ID));
+
+      setTodos(todosFromServer);
+    } catch {
+      setError('unable to get todos');
+    } finally {
+      setIsdataUpdated(false);
+      setTemptodo(null);
+    }
+  };
+
+  const handleTodoSubmit = async (event: ChangeEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    if (!task.trim().length) {
+      setError("The title can't be empty");
+      setTask('');
+
+      return;
+    }
+
+    setTemptodo({
+      id: 0,
+      userId: USER_ID,
+      title: task,
+      completed: false,
+    });
+    setTask('');
+
+    try {
+      await addTodo({
+        userId: USER_ID,
+        title: task,
+        completed: false,
+      });
+    } catch {
+      setError('unable to add todos');
+      setTemptodo(null);
+    } finally {
+      loadTodos();
+      setActiveIds([0]);
+    }
```

Comment:
I don't think that you need loadTodos() in the final step. Move it to try. But set temporary todo to null in any case. Check if it'll work.
```suggestion
      });
      loadTodos();
    } catch {
      setError('unable to add todos');
    } finally {
      setTemptodo(null);
      setActiveIds([0]);
    }
```